onebusaway-android/src/main/java/org/onebusaway/android/ui/FeedbackActivity.java

```diff
@@ -0,0 +1,168 @@
+import android.widget.Toast;
+
+import org.apache.commons.io.FileUtils;
+import org.onebusaway.android.R;
+import org.onebusaway.android.app.Application;
+import org.onebusaway.android.nav.NavigationService;
+import org.onebusaway.android.nav.NavigationServiceProvider;
+import org.onebusaway.android.util.PreferenceUtils;
+
+import java.io.File;
+import java.io.IOException;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import static org.onebusaway.android.nav.NavigationService.LOG_DIRECTORY;
+
+public class FeedbackActivity extends AppCompatActivity {
+
+    public static final String TAG = "FeedbackActivity";
+
+    private String mAction = null;
+    private ImageButton dislikeButton;
+    private ImageButton likeButton;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_feedback);
+        setTitle(getResources().getString(R.string.feedback_label));
+
+        Intent intent = this.getIntent();
+        CheckBox sendLogs = (CheckBox) findViewById(R.id.feedback_send_logs);
+        if(intent != null){
+            mAction = intent.getExtras().getString("CallingAction");
+            if(mAction.equals("no")){
+                Log.d(TAG, "Thumbs down tapped");
+                dislikeButton = findViewById(R.id.ImageBtn_Dislike);
+                dislikeButton.setSelected(true);
+            }
+            else if(mAction.equals("yes")){
+                Log.d(TAG, "Thumbs up tapped");
+                likeButton = findViewById(R.id.ImageBtn_like);
+                likeButton.setSelected(true);
+            }
+        }
+
+        if(Application.getPrefs()
+                .getBoolean(getString(R.string.preferences_key_user_share_logs), true)) {
+            sendLogs.setChecked(true);
+        }
+        else {
+            sendLogs.setChecked(false);
+        }
+
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        // Inflate the menu; this adds items to the action bar if it is present.
+        getMenuInflater().inflate(R.menu.report_issue_action, menu);
+        return true;
+    }
+
+   @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+       if (item.getItemId() == R.id.report_problem_send) {
+           submitFeedback();
+           finish();
+       }
+        return super.onOptionsItemSelected(item);
+    }
+
+    private void submitFeedback() {
+        PreferenceUtils.saveBoolean(NavigationServiceProvider.FIRST_FEEDBACK, false);
+        Log.d(TAG, "First Feedback : " + String.valueOf(Application.getPrefs()
+                .getBoolean(NavigationServiceProvider.FIRST_FEEDBACK, true)));
+        String feedback = ((EditText) this.findViewById(R.id.editFeedbackText)).getText().toString();
+        if (Application.getPrefs()
+                .getBoolean(getString(R.string.preferences_key_user_share_logs), true)) {
+            moveLog(feedback);
+        }
```

Comment:
For all of the IF statements, could you please move the `else` onto the same line as the preceding bracket?

`} else {`

Reply:
Modified the codes to move the else onto the same line.